projects/1.18/assets/lavatech/lavaplus/lang/zh_cn.json

```
@@ -0,0 +1,179 @@
+
+  "item.lavaplus.coppersword": "青铜剑",
+  "message.coppersword":"一把耐用而锋利的剑，比铁造成的伤害更大，但由于它的重量，挥动速度要慢得多。",
+
+  "item.lavaplus.infernalboots": "狱火靴",
+  "message.infernalboots":"非常结实耐用的靴子，可以增加你的最大生命值。",
+
+  "block.lavaplus.crate": "木箱",
+  "message.crate":"一个用于存储的小箱子。",
+
+  "block.lavaplus.blockburner": "方块发电机",
+  "message.blockburner":"将方块转化为能量，每块产生160 FE。",
+
+  "block.lavaplus.inductionfurnace": "感应炉",
+  "message.inductionfurnace":"将原矿石块熔炼成锭。需要熔岩喷口的热量。",
+
+  "block.lavaplus.healer": "治疗者",
+  "message.healer":"自动治疗其上方的玩家，每10游戏刻使用1000 FE恢复半颗心。",
+
+
+  "block.lavaplus.machineframe": "机器框架",
+  "message.machineframe":"用于制作复杂的机器，或者装饰。",
+
+  "block.lavaplus.reinforcedframe": "加固框架",
+  "message.reinforcedframe":"用于游戏末期的更坚固、更耐用的机器框架。",
+
+  "block.lavaplus.heatedmagmablock": "过热岩浆块",
+  "message.heatedmagmablock":"用于加热熔岩喷口，提供两倍于熔岩源的热量",
+
+  "block.lavaplus.lavavent": "熔岩喷口",
+  "message.lavavent":"用于为基于热的机器提供动力。通过围绕熔岩源、过热岩浆块或电加热器来增加热量，最多4个。更多的热量意味着机器运转速度更快，但能源消耗也更快。在其上放置机器。",
+
+  "block.lavaplus.coalinfuser": "煤炭再处理器",
+  "message.coalinfuser":"将熔岩注入煤炭，使其燃烧时间更长。",
+
+  "block.lavaplus.benblock": "Ben 方块",
+  "message.benblock":"可能是，也可能不是爆炸性装置",
+
+  "block.lavaplus.lavagenerator": "热能发电机",
+  "message.lavagenerator":"在被熔岩喷口加热时产生能量。每级热量产生40 FE/t",
+
+  "block.lavaplus.copperdrill": "铜钻头",
+  "message.copperdrill":"提供能量时能在下方1x1区域内钻孔。由于铜钻头本身硬度不高，钻头无法开采深板岩。",
+
+  "block.lavaplus.hydroponics": "生长室",
+  "message.hydroponics":"在有电的情况下能在白天使其中植物生长。",
+
+  "block.lavaplus.basicframe": "基础框架",
+  "message.basicframe":"用于基础机器的原始框架。",
+
+  "block.lavaplus.basicconduit": "基础管道",
+  "message.basicconduit":"一个简单的木制能量导管。不能放在其他基础管道旁边。",
+
+  "block.lavaplus.alloysmelter": "合金炉",
+  "message.alloysmelter":"将锭熔炼成复杂合金，如青铜和铁合金。以煤为燃料。",
+
+  "block.lavaplus.conduit": "能量管道",
+  "message.conduit":"用于在机器之间以2000 FE/t传输能量。",
+
+  "block.lavaplus.battery": "电容",
+  "message.battery":"用于储存能量。只有在给予红石信号时才会输出。",
+
+  "block.lavaplus.centrifuge": "熔岩离心机",
+  "message.centrifuge":"将熔岩分离成基础材料，生产铁、石英、沙子和其他材料",
+
+  "block.lavaplus.compressor": "加热压缩机",
+  "message.compressor":"压缩材料和方块，将它们变成更有价值的物品。",
+
+  "block.lavaplus.upgrader": "升级机",
+  "message.upgrader":"使用狱火锭将下界合金工具和盔甲升级。还可以用下界合金锭和熔岩砖制造狱火锭。",
+
+  "block.lavaplus.heater": "电加热器",
+    "message.heater":"使用60 FE/t为熔岩喷口产生热量。产生与4个熔岩源或2个过热岩浆块一样多的热量。",
+
+  "block.lavaplus.crank": "手摇曲柄",
+  "message.crank":"产生18 FE/t的手动曲柄。",
+
+  "block.lavaplus.watermill": "水车",
+  "message.watermill":"被流动的水包围时产生能量。每边产生5 FE/t，最高可达20 FE/t",
+
+  "block.lavaplus.carboninfuser": "碳灌注机",
+  "message.carboninfuser":"这台机器使用纯碳（煤）来生产各种材料，以及将其他形式的碳，如木炭和生物物质净化成煤。",
+
+  "block.lavaplus.simplebeacon": "机械信标",
+  "message.simplebeacon":"使用50 FE/t在10格半径内产生药水效果，一次只能对一名玩家施加。",
+
+"block.lavaplus.garden": "水培生长室",
+  "message.garden":"进阶版的生长室，水培花园使用30 FE/t来种植植物。不像它的前代，它不消耗种子，也不需要阳光",
+
+  "block.lavaplus.fuelprocessor": "营养燃料处理室",
+  "message.fuelprocessor":"顶部需要2个燃料处理器组件来完成多方块结构。根据食物价值将食物转化为营养燃料电池。",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "message.fuelprocessor":"顶部需要2个燃料处理器组件来完成多方块结构。根据食物价值将食物转化为营养燃料电池。",
```

食物价值？

原文：**Needs 2 Fuel Processor components on top to complete the multiblock. Turns food into Nutritional Fuel Cells based on their food value. **